components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa/src/Publisher/Models/OpcAuthenticationMode.cs

```diff
@@ -0,0 +1,21 @@
+﻿// ------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All rights reserved.
+//  Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
+
+namespace Microsoft.Azure.IIoT.OpcUa.Publisher.Config.Models {
+    using System.Runtime.Serialization;
+
+    /// <summary>
+    /// Enum that defines the authentication method
+    /// </summary>
+    [DataContract]
+    public enum OpcAuthenticationMode {
```

Comment:
>OpcAuthenticationMode [](start = 16, length = 21)

would be good to follow naming convention in the opc stack, e.g.
public enum UserTokenType
    {
        /// <remarks />
        [EnumMember(Value = "Anonymous_0")]
        Anonymous = 0,

        /// <remarks />
        [EnumMember(Value = "UserName_1")]
        UserName = 1,

        /// <remarks />
        [EnumMember(Value = "Certificate_2")]
        Certificate = 2,

        /// <remarks />
        [EnumMember(Value = "IssuedToken_3")]
        IssuedToken = 3,
    }

Reply:
Comments will be fixed in refactoring.